website/docs/r/db_instance_automated_backups_replication.markdown

```diff
@@ -87,6 +87,14 @@ In addition to all arguments above, the following attributes are exported:
 
 * `id` - The Amazon Resource Name (ARN) of the replicated automated backups.
 
+## Timeouts
+
+`aws_db_instance_automated_backups_replication` provides the following [Timeouts](https://www.terraform.io/docs/configuration/blocks/resources/syntax.html#operation-timeouts)
+configuration options:
+
+- `create` - (Default `75m`) How long to wait before RDS successfully starts replication of automated backups to a different AWS Region.
+- `update` - (Default `75m`) How long to wait before RDS completes updating existing replication of automated backups to a different AWS Region.
```

Comment:
This doesn't align with
```go
	InstanceAutomatedBackupsReplicationCreateTimeout = 75 * time.Minute
	InstanceAutomatedBackupsReplicationDeleteTimeout = 75 * time.Minute
```

One is `create` and `update`; the other is _create_ and _**delete**_.

Reply:
Thanks @sftim, sharp. Updated.
It should be only _Create_ and _Delete_ as there's no update possible.